settingup.tex

```diff
@@ -178,6 +177,6 @@ \section{The Intro Disk}
 
 \section{The Cursor}
 
-The flashing square underneath the \screentext{READY} prompt\index{READY prompt} is called the cursor. The cursor indicates that the computer is ready to accept input. Pressing keys on the keyboard will print their respective characters onto the screen. The characters will be printed at the current cursor position, and the cursor will advance to the next position after every key press.
+The flashing square underneath the \texttt{READY} prompt\index{READY prompt} is called the \emph{cursor}. The cursor indicates that the computer is ready to accept input. Pressing keys on the keyboard will print their respective characters onto the screen. The characters will be printed at the current cursor position, and the cursor will advance to the next position after every key press.
```

Comment:
I mostly agree that we need to use the C64 typeface sparingly (much more sparingly than in previous drafts) and the latest Style Guide reflects that, so I appreciate the attention. I had left the few mentions of the READY prompt styled this way as a flourish, like an icon. I don't mind switching to `\texttt{READY}` but we should do so consistently, and also decide whether it should use the period or not. (We can also add this to the style guide, if it's not there yet.) Any preference?

Reply:
I agree. I think that particular one is the one I used to compare the looks of (but forgot to revert). I'm one of those who believe the C64 typeface should normally not be used in text, tables etc. It is harder on the eyes and kind of distracts from the reading I think. It should only be used in examples for those rare occasions where the actual pixelised font helps understanding a definition, concept, or similar. But I do get the idea behind it and the retro feel. 😀

Reply:
I will revert to "screentext" in the mean time, but keeping the "emph".